tests/tests.go~

```diff
@@ -0,0 +1,49 @@
+package tests
```

Comment:
This file name is strange.  Contents as well.

Reply:
Sure, this comes from emacs, removing